### DISCUSSION TITLE
Looking for more maintainers

### DISCUSSION THREAD

**Discussion by kigster** [bazelruby/rules_ruby#146] [Announcements]
Are you proficient in Bazel? Have you written custom rules for a new language?

This project desperately needs a couple of additional core maintainers because @yugui is very busy and I've been also totally swamped.

If you'd like to become a maintainer of this project (and be able to eg fix the CI), please email kigster AT gmail and share your interest and qualifications. You must submit one PR to be considered.

We really need your help, so please consider landing a hand if you feel you understand custom bazel rules and some ruby. We can provide ruby expertise (16 years of professional ruby development) but not nearly as much Bazel.

Don't be shy! Giving back to open source is awesome and would look great on your resume 😃

Thanks!

**Comment by alexeagle** [comment]
The Bazel Rules Authors SIG has some funding specifically to help the rules ecosystem remain maintained. Usually the projects it funds are in the github.com/bazel-contrib org however. Do you have a desire to continue operating this bazelruby org, or are you amenable to relocating the project?

**Reply by kigster** [reply]
Very amenable!

**Reply by kigster** [reply]
I just want to stay on as one of the maintainers, but I'd love it if there were more folks involved.
